mesh/meshdb.go

```diff
@@ -223,23 +208,22 @@ func (m *MeshDB) ForBlockInView(view map[types.BlockID]struct{}, layer types.Lay
 	return nil
 }
 
-func (m *MeshDB) layerBlockIds(index types.LayerID) ([]types.BlockID, error) {
-
-	ids, err := m.layers.Get(index.ToBytes())
+func (m *MeshDB) LayerBlockIds(index types.LayerID) ([]types.BlockID, error) {
+	idsBytes, err := m.layers.Get(index.ToBytes())
 	if err != nil {
 		return nil, fmt.Errorf("error getting layer %v from database %v", index, err)
 	}
 
-	if len(ids) == 0 {
+	if len(idsBytes) == 0 {
 		return nil, fmt.Errorf("no ids for layer %v in database ", index)
 	}
 
-	idSet, err := types.BytesToBlockIds(ids)
+	blockIds, err := types.BytesToBlockIds(idsBytes)
```

Comment:
"idSet" name was on purpose to signify that there are no repetitions, why change it ?

Reply:
`Set` was confusing to me. I thought it signified a set data structure (e.g. implemented as a map), so I was surprised to see it was a slice. Maybe I'll change it to `uniqueIds` if you think it's important to signify it here. Maybe it would be even better to change the method name to `UniqueLayerBlockIds` if you think it's important to signify that, since an internal variable name isn't very visible.

I personally would expect `LayerBlockIds` not to return duplicates even without spelling it out, but I don't think any of this is critical. Just let me know what you think is best and I'll do it (including reverting to the original).

Reply:
i dont think its good practice to introduce unnecessary changes just because it makes sense to you, i can understand a typo fix but specific naming fixes can cause variation in convention,
also makes it harder to keep up with changes (more diffs in your pr), this is not to say that we don't need to refactor our code and maybe provide better naming.
do you know if this convention exists in other locations ? what would prevent me from "fixing" this and changing it back because it makes more sense to me ?

Reply:
I agree. I made this change because I thought it was a leftover from changing the data structure (I thought that's what `Set` meant). That's why I suggested to revert the change if you feel that it's a better description of the variable (this answers your last question - I'm fine with you changing it back and actually suggested to do it myself).

I wasn't kidding - just let me know what you think is best here and I'll do it. I didn't ask for your review out of politeness - I really want your opinion.

Reply:
the specific change is really not that important and the new names are fine , just wanted to point out the problem in the process.